ols/constants.py

```diff
@@ -118,9 +118,9 @@ class GenericLLMParameters:
 
 
 # cache constants
-IN_MEMORY_CACHE = "memory"
+CACHE_TYPE_MEMORY = "memory"
```

Comment:
`"in-memory"`?

Reply:
probably not at this moment (summing time), as it would be user-visible change (`olsconfig.yaml` etc.)

Reply:
please no :)

it was in-memory originally, then we changed it to memory because i didn't like how "in-memory" manifested in the user facing config.

tbh i debated nacking this whole PR for that reason, but i appreciated the change from "MemoryConfig" to MemoryCacheConfig" regardless of the change to "InMemory" vs "Memory"